corehq/apps/app_manager/static/app_manager/js/details/case_claim.js

```diff
@@ -402,7 +402,7 @@ hqDefine("app_manager/js/details/case_claim", function () {
                     };
                     return {
                         name: p.name(),
-                        label: p.label().length ? p.label() : p.name(),  // If label isn't set, use name
+                        label: (p.label().length || p.isGroup) ? p.label() : p.name(),  // If label isn't set, use name
```

Comment:
I'm unclear based on the commit message whether this change is allowing a group to have a blank label or disallowing it. I think the former, since if it's a group but `p.label()` has no length, it will use the blank `p.label()` anyway, right?

Reply:
Yes, it allows the group to have a blank label